config-vm.md

```diff
@@ -0,0 +1,51 @@
+# <a name="VirtualMachineSpecificContainerConfiguration" /> Virtual-machine-specific Container Configuration
+
+Virtual-machine-based containers require additional configuration to that specified in the [base spec configuration](config.md).
+
+This **optional** configuration is specified in a "VM" object:
+
+* **`hypervisor`** (object, OPTIONAL) specifies details of the hypervisor that manages the container virtual machine.
+* **`kernel`** (object, REQUIRED) specifies details of the kernel to boot the container virtual machine with.
+* **`image`** (object, OPTIONAL) specifies details of the image that contains the root filesystem for the container virtual machine.
+
+## <a name="HypervisorObject" /> Hypervisor Object
+
+Used by virtual-machine-based runtimes only.
+
+* **`path`** (string, REQUIRED) specifies the host path to the hypervisor used to manage the container virtual machine.
+* **`parameters`** (array of strings, OPTIONAL) specifies an array of parameters to pass to the hypervisor.
+
+## <a name="KernelObject" /> Kernel Object
+
+Used by virtual-machine-based runtimes only.
+
+* **`path`** (string, REQUIRED) specifies the host path to the kernel used to boot the container virtual machine. This is an absolute path on the host.
+* **`parameters`** (array of strings, OPTIONAL) specifies an array of parameters to pass to the kernel.
+* **`initrd`** (string, OPTIONAL) specifies the host path to an initial ramdisk to be used by the container virtual machine.
+
+## <a name="ImageObject" /> Image Object
+
+Used by virtual-machine-based runtimes only.
+
+* **`path`** (string, REQUIRED) specifies the absolute host path to the container virtual machine root image. This image contains the root filesystem that the virtual machine **`kernel`** will boot into, not to be confused with the container root filesystem itself. The latter, as specified by **`path`** from the [Root Configuration](config.md#Root-Configuration) section, will be mounted inside the virtual machine at a location chosen by the virtual-machine-based runtime.
+
+
+## <a name="FullyPopulatedVMExample" /> Example of a fully-populated `VM` object
```

Comment:
There's no precedent for “fully-populated”.  I'd rather have per-`vm`-child-property examples like the other platform-specific sections.  I have no preference between rooted examples (like [the Windows section uses][1]) or relative example (like the [Solaris][2] and [Linux][3] sections use).  A fully-populated example could go [in here][4].  And if you decide to keep a full `vm` example here, I'd just title it “Example” and rely on [section nesting][5] to define the example scope.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config-windows.md#example
[2]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config-solaris.md#example
[3]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config-linux.md#example
[4]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config.md#configuration-schema-example
[5]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/style.md#example